tutorsuperset/templates/superset/apps/pythonpath/openedx_sso_security_manager.py

```diff
@@ -57,13 +119,20 @@ def access_token(self):
         """
         Returns the string access_token portion of the current OAuth token.
         """
-        return self.get_oauth_token().get('access_token')
+        return self.get_oauth_token()
```

Comment:
Please revert this as [described above](https://github.com/openedx/tutor-contrib-superset/pull/44/files#diff-86883c60f8e1c94fe60abff51337290ebb6d04e9fac49f8c6731d7613045f445R86):

```suggestion
        return self.get_oauth_token().get('access_token')
```